server/src/controllers/notification.controller.js

```diff
@@ -0,0 +1,55 @@
+const Notification = require("../models/notification.model")
+const responseHandler = require("../helpers/responseHandler");
+
+const ObjectID = require('mongodb').ObjectID;
+
+const getNotifications = async(req, res) => {

```

Comment:
You can add here description which you mention [here](https://github.com/purnima143/Kurakoo/pull/458/files#diff-cb4a11377e70fceae76797806915f650f162bbb61a18e2cc1883de7ce929d16cR10)